<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="calibrationFileLayout">
     <item>
      <widget class="QLabel" name="calibrationFileLabel">
       <property name="text">
        <string>Calibration file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="calibrationFileLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="calibrationFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="pointsFileLayout">
     <item>
      <widget class="QLabel" name="pointsFileLabel">
       <property name="text">
        <string>Points file:        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pointsFileLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="pointsFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loadLayout">
     <item>
      <spacer name="loadSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="niceLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="graphisLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>480</width>
         <height>339</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>480</width>
         <height>339</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="interactive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="controlsLayout">
       <item>
        <widget class="QComboBox" name="trajectoryComboBox"/>
       </item>
       <item>
        <widget class="QComboBox" name="indexComboBox"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="stepLayout">
         <item>
          <widget class="QLabel" name="stepLabel">
           <property name="text">
            <string>Step</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="stepSpinBox">
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="scaleGroupBox">
         <property name="title">
          <string>Scale</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="xScaleLayout">
            <item>
             <widget class="QLabel" name="xScaleLabel">
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="xScaleSpinBox">
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="yScaleLayout">
            <item>
             <widget class="QLabel" name="yScaleLabel">
              <property name="text">
               <string>y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="yScaleSpinBox">
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="offsetGroupBox">
         <property name="title">
          <string>Offset</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="xOffsetLayout">
            <item>
             <widget class="QLabel" name="xOffsetLabel">
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="xOffsetSpinBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="yOffsetLayout">
            <item>
             <widget class="QLabel" name="yOffsetLabel">
              <property name="text">
               <string>y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="yOffsetSpinBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="runLayout">
     <item>
      <spacer name="runSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
